GeneticAlgorithm.py

```diff
@@ -371,6 +385,29 @@ def initialize(self):
 
             self._assignments.append(newCourseAssignment)
 
+            if (len(GAList) - 1) == gata_lab_counter:
+                gata_lab_counter = 0
+            else:
+                gata_lab_counter += 1
+
+        for cur_lab in unique_labs:
```

Comment:
Go through unique labs and assign TAs to labs taught by faculty as well as TA taught lab.